tests/plugins/colorbutton/_helpers/tools.js

```diff
@@ -0,0 +1,49 @@
+/* exported findInPanel, chooseColorFromDialog, asyncChooseColorFromDialog */
```

Comment:
You are still exposing all global functions in this file, this comment only serves for linting. It's better to just wrap whole code in iffy module and expose needed API as a part of it. Eg see https://github.com/ckeditor/ckeditor4/blob/f1496f45653f1ee1880da493dd066ad3230232b5/tests/plugins/easyimage/_helpers/tools.js